app/main.py

```diff
@@ -1,6 +1,6 @@
 def copy_file(command: str) -> None:
-    origin_name, copy_name = command.split()[1], command.split()[2]
-    if origin_name == copy_name:
+    cp, origin_name, copy_name = command.split()
+    if cp != "cp" or origin_name == copy_name:
         return
```

Comment:
one more task, try to change your code so you won't get raw return, because it's a bad sign

Reply:
ok
I removed `return`